### PR TITLE
paddlepaddle CN notebook

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Added PaddlePaddle Notebook for Chinese customers

### CODE REVIEW

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -110,7 +110,12 @@ nav:
         - TensorFlow Model Zoo: 'tensorflow/tensorflow-model-zoo/README.md'
   - PaddlePaddle:
       - Overview: 'paddlepaddle/README.md'
-      - Load a PaddlePaddle Model: 'jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb'
+      - Facemask detection using PaddlePaddle: 
+          - English: 'jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb'
+          - 中文: 'jupyter/paddlepaddle/face_mask_detection_paddlepaddle_CN.ipynb'
+      - PaddleOCR example:
+          - English: 'jupyter/paddlepaddle/paddle_ocr_java.ipynb'
+          - 中文: 'jupyter/paddlepaddle/paddle_ocr_java_CN.ipynb'
```
Comment (frankfliu) [inline docs/mkdocs.yml:118]: the file name should be paddle_ocr_java_zh.ipynb

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle_CN.ipynb [hunk 2]
```diff
@@ -0,0 +1,365 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 用飛槳+DJL實現人臉口罩辨識¶\n",
```
Comment (frankfliu) [inline jupyter/paddlepaddle/face_mask_detection_paddlepaddle_CN.ipynb:7]: What's the last char?

**Review by goswamig — APPROVED** [review]
How do we make sure that these notebook are still running fine with new DJL releases ?

**Review by stu1130 — APPROVED** [review]

**Review by frankfliu — APPROVED** [review]

**Comment by lanking520** [comment]
> How do we make sure that these notebook are still running fine with new DJL releases ?

We had a weekly notebook test that will run all notebooks

**Comment by goswamig** [comment]
> > How do we make sure that these notebook are still running fine with new DJL releases ?
> 
> We had a weekly notebook test that will run all notebooks

Let's keep it running and I believe we have alarm in case of failure.

**Pull request merged**